be1-go/internal/popserver/handler/root.go

```diff
@@ -0,0 +1,230 @@
+package handler
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"popstellar/crypto"
+	"popstellar/internal/popserver/config"
+	"popstellar/internal/popserver/database"
+	"popstellar/internal/popserver/database/sqlite"
+	"popstellar/internal/popserver/state"
+	"popstellar/message/answer"
+	"popstellar/message/messagedata"
+	"popstellar/message/query/method/message"
+)
+
+const (
+	Root       = "/root"
+	RootPrefix = "/root/"
+	Social     = "/social"
+	Chirps     = "/chirps"
+	Reactions  = "/reactions"
+	Consensus  = "/consensus"
+	Coin       = "/coin"
+	Auth       = "/authentication"
+)
+
+func handleChannelRoot(msg message.Message) *answer.Error {
+	object, action, errAnswer := verifyDataAndGetObjectAction(msg)
+	if errAnswer != nil {
+		return errAnswer.Wrap("handleChannelRoot")
+	}
+
+	switch object + "#" + action {
+	case messagedata.LAOObject + "#" + messagedata.LAOActionCreate:
+		errAnswer = handleLaoCreate(msg)
+	default:
+		errAnswer = answer.NewInvalidMessageFieldError("failed to handle %s#%s, invalid object#action", object, action)
+	}
```

Comment:
wouldn't the intention be clearer as following ? 
```suggestion
	if object == messagedata.LAOObject && action == messagedata.LAOActionCreate {
		errAnswer = handleLaoCreate(msg)
	} else {
		errAnswer = answer.NewInvalidMessageFieldError("failed to handle %s#%s, invalid object#action", object, action)
	}
```

If you take it one step further, you can even unwrap this further as a series of guard clauses:

```suggestion
	if object != messagedata.LAOObject || action != messagedata.LAOActionCreate {
		return answer.NewInvalidMessageFieldError("failed to handle %s#%s, invalid object#action", object, action).Wrap("handleChannelRoot")
	}
	
	errAnswer = handleLaoCreate(msg)
```